src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -318,3 +344,58 @@ def flatten(dd, separator='.', prefix=''):
         telemetry.set_exception(exception=e, fault_type=consts.Error_Flattening_User_Supplied_Value_Dict,
                                 summary='Error while flattening the user supplied helm values dict')
         raise CLIError("Error while flattening the user supplied helm values dict")
+
+
+def get_latest_kubernetes_version():
+    retries = Retry(total=3, backoff_factor=1, status_forcelist=[413, 429, 500, 502, 503, 504])
+    req_session = requests.Session()
+    adapter = TimeoutHTTPAdapter(max_retries=retries)
+    req_session.mount("https://", adapter)
+    req_session.mount("http://", adapter)
+
+    url = "https://api.github.com/repos/kubernetes/kubernetes/releases/latest"
+
+    payload = {}
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    try:
+        response = req_session.request("GET", url, headers=headers, data=payload)
+        if response.status_code == 200:
+            latest_release = json.loads(response.text)
+            return latest_release["tag_name"]
+        else:
+            logger.warning("Couldn't fetch the latest kubernetes stable release information. Response status code: {}".format(response.status_code))
+    except Exception as e:
+        kubernetes_exception_handler(e, consts.Kubernetes_Latest_Version_Fetch_Fault, "Couldn't fetch latest kubernetes release info", error_message="Error while fetching the latest stable kubernetes release", raise_error=False)
```

Comment:
kubernetes_exception_handler cannot  be used here. It is for requests to k8s api-server. We are setting user_fault by default in that. But this seems to be a different scenario

Reply:
okay, will remove it from here.